integrated_channels/canvas/exporters/content_metadata.py

```diff
@@ -24,6 +24,7 @@ class CanvasContentMetadataExporter(ContentMetadataExporter):
         'is_public': 'is_public',
```

Comment:
In the docs of the `transform` method below for this `is_public` param, we claim that it is to set the course to the public index, but after some investigation I'm fairly certain that it is not the `is_public` param but instead the `indexed` param. According to the Canvas API docs `is_public` is "Set to true if course is public to both authenticated and unauthenticated users." Not sure exactly what that does or if it's actually needed. Any clues? Perhaps it can be removed

Reply:
did some discovery- The `is_public` param allows students to view to content of the course. Without the `is_public` param the course is discoverable in the sense that it will show up in the catalog, but no content shows up when navigating to the actual course: ie 
![image](https://user-images.githubusercontent.com/67655836/104645116-a3dc7680-567c-11eb-9d2b-0a1fa50633bf.png)

when the `is_public` parameter is included then the student can view the course content: ie 
![image](https://user-images.githubusercontent.com/67655836/104645200-c40c3580-567c-11eb-83e1-e09b385b5cd6.png)

So I feel as though we want to keep this parameter but perhaps a doc change so it more closely reflects the actual behavior?

Reply:
yes indeed their doc sucks! this was admitted by canvas folks themselves hence I pasted this note in the ticket for easy access

```
[2] We find we are unable to set the following settings automatically when creating a course:
        "Include this course in the public course index": true (is false by default)
        Context: It may be a great ability to have to auto publish and auto include in public index. (unless the customer does not want us to do so)
I'm not sure if there's a specific product reason why this isn't documented – but you can do this by including the `course[indexed]` parameter when creating a course, and setting the value to '1'
```